crates/bevy_ecs/src/schedule/condition.rs

```diff
@@ -893,6 +894,17 @@ pub mod common_conditions {
         move |query: Query<(), With<T>>| !query.is_empty()
     }
 
+    /// Generates a [`Condition`](super::Condition)-satisfying closure that returns `true`
+    /// if there are any entities with the given component type removed.
```

Comment:
Not completely clear to me, I think the following sounds better:
```suggestion
    /// if there are any entity with a component of the given type removed.
```